src/signals/incident/components/form/PlainText/PlainText.test.tsx

```diff
@@ -123,52 +128,52 @@ describe('Form component <PlainText />', () => {
     })
 
     it('should render plain text info correctly', () => {
-      const props = getProps({
-        ...metaProps,
+      const customMeta = {
+        ...defaultProps.meta,
         type: 'info',
-      })
+      }
 
-      render(withAppContext(<PlainText {...props} />))
+      render(withAppContext(<PlainText {...defaultProps} meta={customMeta} />))
 
       expect(screen.getByTestId('plain-text')).toBeInTheDocument()
-      expect(screen.getByText(props.meta.value)).toBeInTheDocument()
+      expect(screen.getByText(defaultProps.meta.value)).toBeInTheDocument()
```

Comment:
yep good distinction to use default and custom props, although props instead of customProps, should be more like the other tests